model/file/registry.yaml

```diff
@@ -111,6 +111,22 @@ groups:
           The user ID (UID) or security identifier (SID) of the file owner.
         stability: experimental
         examples: ["1000"]
+      - id: file.origin_referrer_url
+        type: string
+        brief: >
+          The URL of the webpage that linked to the file.
+        note: > 
+          This information is inteded to be retrieved from the Mark of the Web (NTFS Zone.Identifer ADS Stream) Note that the URL might contain sensitive information.
+        stability: experimental
+        examples: ['https://example.com']     
```

Comment:
maybe makes sense to have more descriptive examples for both fields? like `http://example.com/article1.html` and  `http://example.com/imgs/article1_img1.jpg` to connect both attributes to each other? And also add a note about second attribute?

Are they always connected to each other? Can one be present without another one?